Header header
string monitor_name
string behaviour
string error_code
string fault_msg
int32 criticality
int32 technicality